src/serde_helpers.rs

```diff
@@ -118,6 +118,60 @@ macro_rules! deserialize_variant {
 /// );
 /// ```
 ///
+/// You don't have to provide all the enumeration variants and can use `_` to put every undefined tag into an enumeration variant like so:
+/// # Example
+///
+/// ```
+/// # use pretty_assertions::assert_eq;
+/// use quick_xml::de::from_str;
+/// use quick_xml::impl_deserialize_for_internally_tagged_enum;
+/// use serde::Deserialize;
+///
+/// #[derive(Deserialize, Debug, PartialEq)]
+/// struct Root {
+///     one: InternallyTaggedEnum,
+///     two: InternallyTaggedEnum,
+///     three: InternallyTaggedEnum,
+/// }
+///
+/// #[derive(Debug, PartialEq)]
+/// enum InternallyTaggedEnum {
+///     NewType(Newtype),
+///     Other,
+/// }
+///
+/// #[derive(Deserialize, Debug, PartialEq)]
+/// struct Newtype {
+///     #[serde(rename = "@attribute")]
+///     attribute: u64,
+/// }
+///
+/// // The macro needs the type of the enum, the tag name,
+/// // and information about all the variants
+/// impl_deserialize_for_internally_tagged_enum!{
+///     InternallyTaggedEnum, "@tag",
+///     ("NewType"    => NewType(Newtype)),
+///     (_  => Other),
```

Comment:
```suggestion
///     ("NewType" => NewType(Newtype)),
///     (_ => Other),
```
In the first example extra spaces used to make table layout, which is redundant here